client/components/location-form.jsx

```diff
@@ -88,25 +89,26 @@ export default class LocationForm extends React.Component {
         <h1 className="page-title">Map a Route</h1>
         <div className="input-div">
           <label htmlFor="startA" className="location-label">Start (A)</label>
-          <input type="text" value={this.state.A} onChange={this.handleChangeA}
+          <input type="text" value={this.state.A.name} onChange={this.handleChangeA}
             className="location-input" ref={this.autocompleteRefA} id="startA"
-            required={true} />
+            required={false} />
         </div>
         <div className="input-div">
           <label htmlFor="stopB" className="location-label">Stop (B)</label>
-          <input type="text" value={this.state.B} onChange={this.handleChangeB}
+          <input type="text" value={this.state.B.name} onChange={this.handleChangeB}
             className="location-input" ref={this.autocompleteRefB} id="stopB"
-            required={true} />
+            required={false} />
         </div>
         <div className="input-div">
           <label htmlFor="stopC" className="location-label">Stop (C)</label>
-          <input type="text" value={this.state.C} onChange={this.handleChangeC}
+          <input type="text" value={this.state.C.name} onChange={this.handleChangeC}
             className="location-input" ref={this.autocompleteRefC} id="stopC"
-            required={true} />
+            required={false} />
```

Comment:
If an input is not required, you can omit the `required` prop. `required` is a boolean prop, so its presence implies `true`

```jsx
<input type="text" required />
```

Reply:
Ahh yes, I had it set to true but turned it off for testing purposes. I will just remove it, for now, to make testing easier!